cypress/e2e/cloud/communityTemplates.test.ts

```diff
@@ -246,12 +246,15 @@ describe('Community Templates', () => {
       cy.go('back')
     })
 
-    // this behavior should be cleaned up
-    it.skip('takes you to github readme when you click on the Community Templates button', () => {
+    it('takes you to github readme when you click on the Community Templates button', () => {
```

Comment:
fyi @zoesteinkamp - this is a better solution than what we were doing (i.e. skipping the test)